content/blog/2023/12/2023-12-05-google-summer-of-code-a-call-for-mentors.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "[Google Summer of Code 2024] A Call for Mentors"
+tags:
+- gsoc2024
+- community
+- contribute
+authors: 
+- krisstern
+- alyssat
+- gounthar
+- jmMeessen
+description: >
+  A short summary to recruit gsoc mentors.
+opengraph:
+  image: /images/post-images/2023/12/gsoc-call-for-mentors.png
+---
+
+image:/images/post-images/2023/12/gsoc-call-for-mentors.png[Google Summer of Code call for mentors., role=center, float=center]
+
+
+We are happy to announce that Jenkins is preparing to participate in its ninth (9th) year in Google Summer of Code (GSoC).
```

Comment:
🤔 I don't come on the same count. Based on the list of years Jenkins participated (bottom of the page), I count 7 participation to this date. So this would be our 8th participation.
A detail and can be fixed afterwards.